lib/puppet_x/puppetlabs/iis/powershell_manager.rb

```diff
@@ -169,7 +169,7 @@ def self.create_event(name, manual_reset = false, initial_state = false)
       WAIT_TIMEOUT = 0x00000102
       WAIT_FAILED = 0xFFFFFFFF
 
-      def self.wait_on(wait_object, timeout_ms = 50)
+      def self.wait_on(wait_object, timeout_ms = 1000)
```

Comment:
It actually shouldn't be messed with, the pain-point is the debug messages not the timing. IMHO the change should be removed from this PR as it was a red herring and we add another PR to discuss removing the debug message. The utility of the message stems from wanting to 'see' some output interactively so that the user can be sure something is happening, without this message there is no indication anything is happening.